doc/src/manual/arrays.md

```diff
@@ -528,10 +528,8 @@ applied to arrays (and combinations of arrays and scalars) as a
 operations like `<`, *only* the `.<` version is applicable to arrays.)
 
 1. Unary arithmetic -- `-`, `+`, `!`
-2. Binary arithmetic -- `+`, `-`, `*`, `/`, `\`, `^`, `.^`, `div`, `mod`
+2. Binary arithmetic -- `+`, `-`, `*`, `/`, `\`, `^`, `.^`
```

Comment:
does `.^` need to be listed separately, when dot operators are described in the paragraph above?